test/test_base.py

```diff
@@ -29,7 +29,7 @@
 # Naming this variable "test_table" triggers the following bug: https://github.com/pytest-dev/pytest/issues/7378
 # noinspection PyTypeChecker
 mock_table = Table(
-    None, "mock_table",
+    Database(PyKustoClient('mock_cluster', fetch_by_default=False), 'mock_database', fetch_by_default=False), "mock_table",
```

Comment:
is this change related to the typing fixes?

Reply:
Yes, the database variable in not `Optional`, so we can't provide `None` here